dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "DevOps-MBSE/aac-program-team"
+    labels:
```

Comment:
Labels: For the dependency configurations be sure to use the dependencies label. For security configuration be sure to use security label. So that our change log will be properly populated.

Reply:
I dont see a way to have two different labels for the same package ecosystem.  Security updates would be configured in a group within a package ecosystem.